src/esp/bindings/SensorBindings.cpp

```diff
@@ -53,14 +57,16 @@ void initSensorBindings(py::module& m) {
       .value("NONE", SensorType::None)
       .value("COLOR", SensorType::Color)
       .value("DEPTH", SensorType::Depth)
-      .value("SEMANTIC", SensorType::Semantic);
+      .value("SEMANTIC", SensorType::Semantic)
+      .value("AUDIO", SensorType::Audio);
 
   py::enum_<SensorSubType>(m, "SensorSubType")
       .value("NONE", SensorSubType::None)
       .value("PINHOLE", SensorSubType::Pinhole)
       .value("ORTHOGRAPHIC", SensorSubType::Orthographic)
       .value("FISHEYE", SensorSubType::Fisheye)
-      .value("EQUIRECTANGULAR", SensorSubType::Equirectangular);
+      .value("EQUIRECTANGULAR", SensorSubType::Equirectangular)
+      .value("IMPULSERESPONSE", SensorSubType::ImpulseResponse);;
```

Comment:
Hmm, wasn't envisioning subtype to be used for AudioSensor's too, but it probably should be. Wonder if there is a better way to ensure invalid states aren't selected. Pinhole Audio etc... Color ImpulseResponse.

Reply:
I check that here : https://github.com/facebookresearch/habitat-sim/blob/sanchitgarg/audio-sensor/src/esp/sensor/AudioSensor.cpp#L31
Would this catch the issue?

Im assuming in the future, we can have more subTypes specific to other audio algorithms.